packages/mask/background/database/avatar-cache/avatar.ts

```diff
@@ -29,6 +35,10 @@ export const queryAvatarDataURL = memoizePromise(
 export async function storeAvatar(identifier: IdentifierWithAvatar, avatar: ArrayBuffer | string): Promise<void> {
     if (identifier instanceof ProfileIdentifier && identifier.isUnknown) return
     try {
+        if (hasNativeAPI) {
+            await nativeAPI?.api.store_avatar({ identifier: identifier.toText(), avatar: avatar as string })
```

Comment:
You didn't handle the ArrayBuffer type

Reply:
`storeAvatar` used with ArrayBuffer is only used in the JS dashboard to set an avatar for the persona, while this is unreachable on our mobile app, we could just ignore the ArrayBuffer type param IMO.
Check the avatar type, do nothing with ArrayBuffer with some comments would be better here